stix_shifter_modules/demo_template/stix_transmission/ping_connector.py

```diff
@@ -0,0 +1,26 @@
+from stix_shifter_utils.modules.base.stix_transmission.base_ping_connector import BasePingConnector
```

Comment:
can we move all *_connector.py method into one `connector.py` file to keep it as simple as possible?